tensorflow/core/kernels/dequantize_op.cc

```diff
@@ -70,12 +70,6 @@ class DequantizeOp : public OpKernel {
                   errors::InvalidArgument("Mode string must be 'MIN_COMBINED',"
                                           " 'MIN_FIRST', or 'SCALED', is '" +
                                           mode_string + "'"));
-    } else {
```

Comment:
Are you sure this isn't breaking something on CPU (e.g. without onednn)?

Reply:
Thanks! Reverted the change.